osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -518,7 +520,8 @@ private void contentIn(double delayBeforeSideDisplays = 0)
                     .ScaleTo(1)
                     .FadeInFromZero(500, Easing.OutQuint);
 
-                this.Delay(quick_restart_initial_delay).Schedule(() => BackButtonVisibility.Value = true);
+                quickRestartBackButtonRestore?.Cancel();
+                quickRestartBackButtonRestore = Scheduler.AddDelayed(() => BackButtonVisibility.Value = true, quick_restart_initial_delay);
```

Comment:
i'm sure you're about to be *overjoyed* with what i have to say here, but this don't look that great (shows the old back button when it kicks in):

https://github.com/user-attachments/assets/c681a77e-a15a-49f8-99d1-2b60085d5caa

given fixing this would invariably require plunging a hand into the innards of the screen footer beast, i think i could be okay with pretending i saw nothing, but i'm not sure that you will be

Reply:
I'm aware yeah. Best effort for now.